### ISSUE TITLE
Loft allows you to put order on more than one truck stop

### ISSUE THREAD

**Issue by kentbower** [kbower/tickettest1#2226]
_Imported from trac ticket 2226.
Created by **kbower**
Opened in trac: Mar 14, 2013, 5:29:34 PM
Last modified in trac: Mar 15, 2013, 10:02:43 AM_
***
_by kbower:_
surprised no one tested this or reported it.

**Comment by kentbower** [comment]
_**kbower** trac comment on Mar 14, 2013, 5:34:12 PM:_

Not sure why the return was added to skip this check.
```python
    def _validate(self):
        # REFACTOR.. fix this .. don't we need this check?
        return
        if self.adding:
            # check that this stop's order doesn't already have a stop
            if DBSession.query(TruckStop)\
                    .filter_by(orderid=self.orderid)\
                    .filter(TruckStop.truckid!=self.truckid)\
                    .all():
                
                
                
                # REFACTOR
                
                
                self.error()
        # Make sure either orderid or transfer fks are populated, not both.
```

**Comment by kentbower** [comment]
_**kbower** trac comment on Mar 14, 2013, 6:09:32 PM:_

(Changeset [[3910]](../commit/3a399171a919c9f528630705035737061c812e06))
* fix warning filter that was preventing raising sqlalchemy warnings as exceptions, re #2226

**Comment by kentbower** [comment]
_**sshah** trac comment on Mar 15, 2013, 10:02:43 AM:_

(Changeset [[3912]](../commit/8269cd54910b5b36bcd71050bcd425c87d147d8f))
* Fixes #2226 Disallow adding order on more than one truck stop

**Comment by kentbower** [comment]
_**sshah** trac status change on Mar 15, 2013, 10:02:43 AM:_ **closed**
